warrior/Framework/Utils/encryption_utils.py

```diff
@@ -54,13 +51,18 @@ def get_key(encoded_key):
             MYFILE = Tools.__path__[0]+os.sep+"admin"+os.sep+'secret.key'
             with open(MYFILE, 'r') as myfileHandle:
                 encoded_key = myfileHandle.read()
+            if not encoded_key:
+                print_warning("Could not find the key  in Tools/Admin/secret.key!"
+                              " use ./Warrior -encrypt anything -secretkey sixteenlenstring")
         except IOError:
-            print_error("Could not find the secret.key file in Tools/Admin!")
-    try:
-        IV = Random.new().read(AES.block_size)
-        CIPHER = AES.new(base64.b64decode(encoded_key), AES.MODE_CFB, IV)
-    except Exception as e:
-        print_exception("Some problem occured: {0}".format(e))
+            print_warning("Could not find the secret.key file in Tools/Admin!"
+                          " use ./Warrior -encrypt anything -secretkey sixteenlenstring")
+    if encoded_key:
+        try:
```

Comment:
Line 61 to 65 can be moved into the ```else``` block of ```try-except-else```. This will make the code cleaner and more readable.

Reply:
done